scripts/bumpver.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python3
```

Comment:
I would recommend:
`#!/bin/env python` instead to avoid issues with virtual environments.

It shouldn't really make much of difference with the current state of the script (no external libs), but in case things change.

Reply:
Fixed.

Reply:
I, on the other hand, prefer `#!/usr/bin/python3` because `#!/bin/env python` says, "Run it with any python interpreter you have specified in your environment.  And that is not what you test and intend to do.